supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -379,6 +379,28 @@ NOTE: When referring generally to other Kubernetes components, such as pods, nod
 
 *See also*:
 
+[discrete]
+[[operator-hub]]
+==== image:images/yes.png[yes] OperatorHub (noun)
+*Description*: In red Hat OpenShift, the "OperatorHub" is a central location where you can find a wide array of useful Operators built by the community.
```

Comment:
Capitalize, italics, and remove "the".

Also, OperatorHub doesn't just house community operators. So suggest removing that part and replacing with something generic or different.

```suggestion
*Description*: In Red Hat OpenShift, _OperatorHub_ is a central location where you can find a wide array of useful Operators to install.
```